### PR TITLE
COMMERCE-4876 The view details on the mini-cart does not work the first time you try it

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/index.js [hunk 1]
```diff
@@ -13,7 +13,12 @@
  */
 
 import getJsModule from '../../../utilities/modules';
-import {DISCOUNT_LEVEL_PREFIX, ORDER_UUID_PARAMETER} from './constants';
+import {
+	DISCOUNT_LEVEL_PREFIX,
+	DEFAULT_ORDER_DETAILS_PORTLET_ID,
+	ORDER_DETAILS_ENDPOINT,
+	ORDER_UUID_PARAMETER
```
Comment (dsitu) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/util/index.js:20]: I think sf is complaining about this ahha, sorting should fix it.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**